lang-lisp/chezscheme/autobuild/build

```diff
@@ -0,0 +1,13 @@
+abinfo "Configuring..."
+# parameters were taken from Arch Linux
```

Comment:
I would sincerely hope that you understand what these parameters mean - and this comment is not giving me a lot of confidence.